Podfile.lock

```diff
@@ -1,83 +1,101 @@
 PODS:
-  - Firebase/Analytics (7.5.0):
+  - Firebase/Analytics (8.3.0):
```

Comment:
I don't see a connection between task and updates, if those are not needed to complete the task please make a separate PR